dataflux_pytorch/benchmark/checkpointing/multinode/README.md

```diff
@@ -57,3 +57,20 @@ To run the script use the following command.
 ```shell
 python3 -m dataflux_pytorch.benchmark.checkpointing.multinode.train
 ```
+
+### Distributed Async Checkpoint Save
+
+Additionally, you can make your distributed checkpointing save calls asynchronous by using the `AsyncDatafluxFSDPStrategy`. Under the hood, this will leverage the [torch.distributed.checkpoitn.async_save](https://pytorch.org/docs/stable/distributed.checkpoint.html#torch.distributed.checkpoint.state_dict_saver.async_save) to launch the checkpoint save in a separate thread.
+
+This separate demo introduces a new env var to increase the number of training steps between checkpoint saves, which results in more training cycles between checkpoint saves, as well as a var to select using distributed `save` or `async_save` behavior.
```

Comment:
We can shorten this to (and also be specific about the var type) "This separate demo introduces a new env var to increase the number of training steps between checkpoint saves, as well as an env var to select distributed `save` or `async_save` behavior.
Comment"

Reply:
Was your suggestion to remove line 63? Or update line 65 to match the updated env vars?

Reply:
Match the env vars and remove some unnecessary text.